Resources/Maps/reach.yml

```diff
@@ -11645,13 +11642,6 @@ entities:
     - type: Transform
       pos: 8.5,-8.5
       parent: 2
-- proto: MaterialReclaimer
-  entities:
-  - uid: 1662
-    components:
-    - type: Transform
-      pos: 10.5,-8.5
-      parent: 2
```

Comment:
This isn't a circuit imprinter

Reply:
Guess it's from this PR
https://github.com/space-wizards/space-station-14/pull/30802

Reply:
yeah I didn't remove that. I guess it might have been removed automatically because of the recycler PR